src/animation/AnimationMixer.js

```diff
@@ -6,6 +6,10 @@ import { PropertyMixer } from './PropertyMixer.js';
 import { AnimationClip } from './AnimationClip.js';
 import { NormalAnimationBlendMode } from '../constants.js';
 
+
+const _controlInterpolantsResultBuffer = /*@__PURE__*/ new Float32Array( 1 );
```

Comment:
I've thought the `/*@__PURE__*/` annotation is only needed for custom types. How does it effect tree-shaking when using it with built-in types like `Float32Array`?

Reply:
It works on any javascript variable or function call apparently, see the `treeshake.annotations` section:

https://rollupjs.org/guide/en/#treeshake